features/user_can_rate_recipes.feature

```diff
@@ -0,0 +1,32 @@
+Feature: User can rate recipes
+  As a user
+  In order to evaluate recipes
+  I would like to be able to rate recipes
+
+Background:
+  Given We have the following user:
+    | email           | password    |
+    | maran@test.com  | 12345678    |
+
+  Given We have the following recipes:
+    | title       	| description                 | ingredients        | directions              | user            |
+    | Meatball-Stew | This will make you sick     | Meat, onion, stuff | Stir it, mix it, eat it | maran@test.com  |
+
+  And I am logged in as "rater@test.com"
```

Comment:
Shouldn't this user be created, before being logged in?

Reply:
Ok, just realized the "I am logged in as {user}" function has been modified to create users. Ignore this.